### ISSUE TITLE
Not working on all devices

### ISSUE THREAD

**Issue by vasilenastas** [daimajia/AndroidSwipeLayout#343]
I implemented v1.20@aar but not working on all devices. http://stackoverflow.com/questions/38394974/android-swipelayout-not-working-on-all-devices

Any one got such problem and found solution for it ?

**Comment by Buckstabue** [comment]
Yes, I've got this problem on my Tele 2 Mini smartphone with android 5.1. The layout cannot be swiped on this device. When I try to swipe, nothing happens for the first second, then the bottom view opens very fast and hides without any animation....

**Comment by Sigvent** [comment]
same question than #203
